pkg/apis/config/v1alpha1/types.go

```diff
@@ -38,6 +38,9 @@ type ControllerConfiguration struct {
 	// HealthCheckConfig is the config for the health check controller
 	// +optional
 	HealthCheckConfig *healthcheckconfigv1alpha1.HealthCheckConfig `json:"healthCheckConfig,omitempty"`
+	// BastionComputeConfig is the config for the Bastion openstack compute
+	// +optional
+	BastionComputeConfig *BastionComputeConfig `json:"bastionComputeConfig,omitempty"`
```

Comment:
i'd rather have a `bastionConfig` and not just limit it to `compute` configurations.

Reply:
committed in fc634219